### DISCUSSION TITLE
cs/algorithm/ds/summary1/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#16] [General]
# cs/algorithm/ds/summary1/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/cs/algorithm/ds/summary1/

**Comment by AnyaReese** [comment]
AOV 网络这里的 latest completion time 是不是应该取 min，否则好像会影响后面任务的完成（？

**Reply by TonyCrane** [reply]
是，写错哩
